docs/rain_stackset_deploy.md

```diff
@@ -43,6 +43,7 @@ rain stackset deploy <template> [stackset] [flags]
       --accounts strings   accounts for which to create stack set instances
   -c, --config string      YAML or JSON file to set additional configuration parameters
   -d, --detach             once deployment has started, don't wait around for it to finish
+  -f, --force              update the stackset without confirmation
```

Comment:
Would -y be a better option to match the deploy command?

Reply:
yup that's more consistent. I fixed it.